modules/nlu/src/backend/engine2/engine2.ts

```diff
@@ -74,6 +76,7 @@ export default class Engine2 {
     const model = await Trainer(input, Engine2.tools, token)
     // TODO handle this logic outside. i.e (distributed)job-service ?
     if (model.success) {
+      debugNLU('successfully finished training')
```

Comment:
In engine1, I moved it from debug to info so we could see if it was training without enabling debug traces, since it displays way too much stuff